README.md

```diff
@@ -13,6 +13,7 @@
 [![codecov](https://codecov.io/gh/eclipse-thingweb/node-wot/branch/master/graph/badge.svg)](https://codecov.io/gh/eclipse-thingweb/node-wot)
 [![Telegram Group](https://img.shields.io/endpoint?color=neon&url=https%3A%2F%2Ftg.sumanjay.workers.dev%2Fnodewot)](https://t.me/nodewot)
 [![Discord](https://img.shields.io/badge/Discord-7289DA?logo=discord&logoColor=white&label=node-wot)](https://discord.gg/JXY2Jzefz3)
+[![Static Badge](https://img.shields.io/badge/Show%20Adoption%20and%20Usage-%2331b8a3ff?logoColor=31b8a3ff)](https://github.com/eclipse-thingweb/node-wot#adopters)
```

Comment:
The changes look good to me :) How about something "Adaptors and users" for the label, though? The the leading "Show..." is a bit of a deviation from the other badges.

Reply:
That is much nicer indeed!